apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -413,6 +412,7 @@ export default class ScriptEditor extends React.Component {
                   name="is_course"
                   type="checkbox"
                   defaultChecked={this.props.isCourse}
+                  disabled={!this.state.familyName}
```

Comment:
What happens if you pick a `familyName`, select this checkbox, then set `familyName` to `(None)`? This checkbox would be disabled and couldn't be unchecked right?

Reply:
Oh good call. Will need to look into that.